docs/views/layout.html

```diff
@@ -16,10 +16,9 @@
     <div class="nhsuk-width-container nhsuk-header__container">
       <div class="nhsuk-header__logo">
         <a class="nhsuk-header__link nhsuk-header__link--service" href="/docs" aria-label="NHS.UK prototype kit home">
-          <svg class="nhsuk-logo nhsuk-logo--white" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" role="presentation" focusable="false" viewBox="0 0 40 16">
+          <svg class="nhsuk-logo nhsuk-logo--white" xmlns="http://www.w3.org/2000/svg" role="presentation" focusable="false" viewBox="0 0 40 16">
```

Comment:
We've removed the `role="presentation"` `and focusable="false"` attributes

Reply:
@mcheung-nhs cheers Michael, I've updated that